### DISCUSSION TITLE
I can't customizes styles

### DISCUSSION THREAD

**Discussion by brenogadelha** [eopeter/flutter_mapbox_navigation#385] [General]
Hey Peter, how are you doing? I'd like some help to customize the theme of the navigation. MapboxCustomManeuverStyle and MapboxCustomTripProgressStyle are not working. The application always takes the primary color for the colors of these styles. Can you help me?

**Comment by brenogadelha** [comment]
Only the primary color is working, but... In dark mode, the color doesn't work in time remaining. Only in light mode.

**Reply by yassineoubihi** [reply]
is it working on gradle 8.0 ??
